com.github.phase1geo.minder.yaml

```diff
@@ -0,0 +1,78 @@
+id: com.github.phase1geo.minder
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+base: io.elementary.BaseApp
+base-version: juno-19.08
+command: com.github.phase1geo.minder
+finish-args:
+  - --filesystem=xdg-documents
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+cleanup:
+  - '.la'
+  - '*.a'
+  - /cache
+  - /include
+  - /lib/pkgconfig
+  - /lib/systemd
+  - /lib/valadoc
+  - /lib/valadoc-0.48
+  - /man
+  - /share/aclocal
+  - /share/devhelp
+  - /share/gir-1.0
+  - /share/gtk-doc
+  - /share/man
+  - /share/vala-0.40
+  - /share/vala-0.48
+  - /share/valadoc
+  - /share/valadoc-0.48
+modules:
+  # We need new Vala for Minder > 1.6 version
+  - name: vala
+    cleanup-platform:
+      - '*'
```

Comment:
Just a note, this feels wrong. Why not `cleanup`?

Reply:
Ah, i'll fix this with the next Minder release. Thanks for review guys!